ambry-tools/src/main/java/com.github.ambry/store/ConsistencyCheckerTool.java

```diff
@@ -39,79 +34,59 @@
  * 2) In the index file boundaries on all replicas in a partition
  */
 public class ConsistencyCheckerTool {
+  private final VerifiableProperties verifiableProperties;
```

Comment:
Yes, the dumpIndexTool needs the verifiable props to generate the config by itself.